packages/entity-browser/src/dev/rest-responses/form_user_detail.json

```diff
@@ -131,7 +131,7 @@
                 "displayType": "READONLY",
                 "children": [
                   {
-                    "name": "relUser_code",
+                    "name": "relUser_code1",
```

Comment:
Because I created a json response mock for the User_code1 entity to load the store of the select box :) I also could have changed the response mock instead of this form definition. But its consistent now.